### PR TITLE
docs: fix search query to include sub pages correctly

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

File: packages/dnb-design-system-portal/src/uilib/search/__tests__/searchQuery.test.ts [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+          title: 'Title 2',
+        },
+        headings: [{ depth: 3, slug: 'heading-3', value: 'Heading 3' }],
+        slug: '/page-1',
+        title: 'Heading 2',
+      },
+    ])
+  })
+
+  it('should use title from siblings, when slug matche', () => {
```
Comment (langz) [inline packages/dnb-design-system-portal/src/uilib/search/__tests__/searchQuery.test.ts:167]:
```suggestion
  it('should use title from siblings, when slug matches', () => {
```

**Review by langz — APPROVED** [review]
Not exactly sure how the two commits is related to each other 🏴‍☠️  
https://github.com/dnbexperience/eufemia/pull/1896/commits/6d328de88a83ba0703f5f52900e44b2286e37ec6 and https://github.com/dnbexperience/eufemia/pull/1896/commits/d12e5a90da21c3f61cef6919720a82f31eed71a0

Doesn't really matter though, we can just keep it like this 🙏

**Comment by tujoworker** [comment]
> Not exactly sure how the two commits is related to each other 🏴‍☠️ [6d328de](https://github.com/dnbexperience/eufemia/commit/6d328de88a83ba0703f5f52900e44b2286e37ec6) and [d12e5a9](https://github.com/dnbexperience/eufemia/commit/d12e5a90da21c3f61cef6919720a82f31eed71a0)

Yeah, [6d328de](https://github.com/dnbexperience/eufemia/commit/6d328de88a83ba0703f5f52900e44b2286e37ec6) could be its own PR. It enables that we actually can run tests inside the portal. There where some setup from before, but it did not work.
This is actually the first unit test there.

**Pull request merged**